src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -101,9 +101,9 @@ public static AppDomain CreateDomain(string friendlyName)
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_AppDomains);
         }
 
-        public int ExecuteAssembly(string assemblyFile) => ExecuteAssembly(assemblyFile, null);
+        public int ExecuteAssembly(string assemblyFile) => ExecuteAssembly(assemblyFile, null!);
 
-        public int ExecuteAssembly(string assemblyFile, string?[]? args)
+        public int ExecuteAssembly(string assemblyFile, string?[] args)
```

Comment:
Fix the `ExecuteAssembly` overload to match

Reply:
Actually, I should've done it in the overload that takes a params. My bad, good catch.

Reply:
Fixed